README.md

```diff
@@ -190,6 +194,8 @@ If you like this project, Please give it a ⭐️
  ## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/), kindly make sure to give credit to Nelson Sakwa